dev-packages/localization-manager/README.md

```diff
@@ -14,6 +14,7 @@
 
 The `@theia/localization-manager` package is used easily create localizations of Theia and Theia extensions for different languages. 
 Its main use case is to extract localization keys and default values from `nls.localize` calls within the codebase.
+Additionally, it uses the [DeepL API](https://www.deepl.com/docs-api) to automatically translate any missing localization values.
```

Comment:
Should we enhance the documentation for the `DeepL` functionality to describe it further, and potentially include examples? We might want to describe how it works.